gabbi/fixture.py

```diff
@@ -86,7 +84,7 @@ def nest(fixtures):
             contexts.append(enter_func())
             exits.append(exit_func)
         yield contexts
-    except Exception:
+    except Exception as e:
```

Comment:
This is going to fail the pep8 tests, see https://github.com/cdent/gabbi/actions/runs/6331200515/job/17327201194?pr=324

Reply:
this was there to made to fixture simpler (simply "raise e"), then got lost in stream of thoughts.